src/python/deepgnn/graph_engine/snark/dispatcher.py

```diff
@@ -85,8 +86,8 @@ def __init__(
             folder (str): Location of graph files.
             parallel (int): Number of parallel process to use for conversion.
             meta (str): Meta data about graph.
-            process (typing.Callable[ [typing.Union[mp.Queue, Connection], mp.Queue, str, int, int, int, DecoderType], None ]): Function to call for processing lines in a file.
-            decoder_type (DecoderType): Decoder which is used to parse the raw graph data file, Supported: json/tsv.
+            process (typing.Callable[ [typing.Union[mp.Queue, Connection], mp.Queue, str, int, int, int, Decoder], None ]): Function to call for processing lines in a file.
```

Comment:
Is it possible to stay consistent between type hints(DecoderType) and docstring(Decoder)?